app/assets/stylesheets/application.css.scss

```diff
@@ -14,3 +14,8 @@
  *= require_tree ../../../vendor/assets/stylesheets
  *= require tracks
  */
+
+.bootstrap {
```

Comment:
Is the use of `.bootstrap` intended to be permanent, or is this to enable a gradual rollout during development?